assets/Zed/js/modules/form-toggler.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+'use strict';
+
+$(document).ready(function() {
+
+    createToggleElements();
+
+    $('.toggle-trigger').each(function (index, selectItem) {
+        var group = $(selectItem).data('toggle-group');
```

Comment:
For better future support suggest to move at lease class names (`toggle-group`, `toggle-item`, `active `) into constants.

Reply:
Fixed.